spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -25,7 +25,7 @@ module XCPretty
 
     it "formats cocoapods errors" do
       @formatter.format_error("The sandbox is not in sync...").should ==
-      "\n#{@formatter.red("⌦  The sandbox is not in sync...")}\n\n"
+      "\n#{@formatter.red("❌  The sandbox is not in sync...")}\n\n"
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer single-quoted strings inside interpolations.